File: korepetynder-spa/src/app/settings/models/requests/tutorLessonRequest.ts
export class TutorLessonRequest {
  cost: number;
  frequency: number | null;
  subjectId: number;
  levelsIds: number[];
  languagesIds: number[];

  constructor(cost: number, frequency: number | null, subjectId: number,
    levelsIds: number[], languagesIds: number[]) {
    this.cost = cost;
    this.frequency = frequency;
    this.subjectId = subjectId;
    this.levelsIds = levelsIds;
    this.languagesIds = languagesIds;
  }
}

File: Korepetynder.Contracts/Requests/Tutors/TutorLessonRequest.cs
namespace Korepetynder.Contracts.Requests.Tutors
{
    public class TutorLessonRequest
    {
        public int Frequency { get; set; }
        public int SubjectId { get; set; }
        public int Cost { get; set; }
        public IEnumerable<int> LevelsIds { get; set; }
        public IEnumerable<int> LanguagesIds { get; set; }

        public TutorLessonRequest(int frequency, int subjectId, IEnumerable<int> levelsIds, IEnumerable<int> languagesIds, int cost)
        {
            Cost = cost;
            Frequency = frequency;
            SubjectId = subjectId;
            LevelsIds = levelsIds;
            LanguagesIds = languagesIds;
        }
    }
}
